apps/deploy-web/src/store/networkStore.ts

```diff
@@ -66,8 +69,42 @@ export const initiateNetworkData = async () => {
   );
 };
 
-const selectedNetwork = atom<Network>(networks[0]);
+const selectedNetworkId = atomWithStorage<Network["id"]>(
```

Comment:
Didn't know about this util, pretty cool!

Reply:
this is actually pretty common pattern. I've seen implementations with every state management lib